package.json

```diff
@@ -24,6 +24,7 @@
   },
   "scripts": {
     "build": "NODE_ENV=production babel src --out-dir lib --source-maps",
+    "check": "eslint src/*.js test/*.js",
     "watch": "babel src --out-dir lib --watch --source-maps",
```

Comment:
Maybe, `lint` command is meaningful.

Also, To define `"lint:fix": "eslint --fix src/*.js test/*.js"` is useful.